templates/html/.partials/schema.html

```diff
@@ -4,6 +4,25 @@
   <h4>{{schemaName}}</h4>
   {{/unless}}
 
+  {{#if schema.items.[0]}}
```

Comment:
Just out of curiosity: doesn't `schema.items.length` work?

Reply:
Cool. Just verified that it works.

Reply:
Awesome! I wasn't sure. Handlebars drives me nuts sometimes.